### DISCUSSION TITLE
Destinos de verano

### DISCUSSION THREAD

**Discussion by wisiwii** [unaizorrilla-githubschool/AccionaNonDevelopers#12] [Ideas]
¿Alguien tiene idea de sitios buenos para veranear en categoría **BBC**? 🌞

**Comment by jsagues89** [comment]
¿Qué significa BBC?

**Reply by itzalin78** [reply]
BBC puede ser una cosa muy fea, no lo busques en Google 😃

**Reply by jorubiop** [reply]
Será bueno *bonito* y **Caro**

**Reply by wisiwii** [reply]
@itzalin78, @jorubiop sí, me refería a **Caro**
